bitrise.yml

```diff
@@ -243,143 +234,6 @@ workflows:
-            else
-                echo "Using Discuss API key for topic: $DISCUSS_API_KEY"
-            fi
-
-            if [ -z "$DISCUSS_USERNAME" ] ; then
-              read -p "Enter Discuss username for topic: " -r
-              envman add --key DISCUSS_USERNAME --value "${REPLY}"
-            else
-                echo "Using Discuss username for topic: $DISCUSS_USERNAME"
-            fi
-    - script:
-        title: Confirm release
-        inputs:
-        - content: |-
-            #!/bin/bash
-            set -e
-
-            echo
-            echo "This action will:"
-            echo "- Set Workflow editor to version $NEW_RELEASE_VERSION"
-            echo "- Create a new release on GitHub"
-            echo "- Create related article in discuss.bitrise.io changelog"
-            echo
-
-            read -p "Are you sure? [y/n] " -n 1 -r
-            echo
-            if [[ ! $REPLY =~ ^[Yy]$ ]] ; then
-              exit 1
-            fi
-    - script:
-        title: Set new version
-        inputs:
-        - content: |-
-            #!/bin/bash
-            set -ex
-
-            sed -i "" -e "s/^  \- RELEASE_VERSION\: .*$/  - RELEASE_VERSION: ${NEW_RELEASE_VERSION}/" bitrise.yml
-            sed -i "" "s/\"version\": \".*\"/\"version\": \"$NEW_RELEASE_VERSION\"/" package.json
-
-            export RELEASE_VERSION="$NEW_RELEASE_VERSION"
-            envman add --key RELEASE_VERSION --value "$RELEASE_VERSION"
-    - script:
-        title: Create changelog
-        inputs:
-        - content: |-
-            #!/bin/bash
-            set -ex
-
-            go get github.com/bitrise-tools/releaseman
-
-            export CI=true
-
-            releaseman create-changelog \
-              --version $NEW_RELEASE_VERSION \
-              --set-version-script "bash _scripts/set_version.sh version/version.go bitrise-plugin.yml"
-    - script:
-        title: Commit & push changes
-        inputs:
-        - content: |-
-            #!/bin/bash
-            set -ex
-
-            git add -A
-            git commit -m "v$NEW_RELEASE_VERSION"
-            git push
-    - script:
-        title: Create binaries for GitHub release
-        inputs:
-        - content: |-
-            #!/bin/bash
-            set -ex
-
-            bitrise run create-binaries
-    - script:
-        title: Release binaries to GitHub
-        inputs:
-        - content: |-
-            #!/bin/bash
-            set -ex
-
-            export GIT_RELEASE_VERSION=$RELEASE_VERSION
-            envman add --key GIT_RELEASE_VERSION --value "$RELEASE_VERSION"
-            bitrise run create-github-release
-    - script:
-        title: Create Discuss release
-        inputs:
-        - content: |-
-            #!/bin/bash
-            set -ex
-
-            export DSICUSS_RELEASE_VERSION=$RELEASE_VERSION
-            envman add --key DSICUSS_RELEASE_VERSION --value "$RELEASE_VERSION"
-            bitrise run create-discuss-release
-
```

Comment:
these are removed from here and moved to https://app.bitrise.io/app/1686da85b5935fd6/workflow_editor#!/workflows?workflow_id=plugin-release